docs/bare-metal/README.md

```diff
@@ -0,0 +1,298 @@
+
+the rest of the options will be discussed below.
+
+## Getting Started with Ansible
+
+
+Add your key to all the machines in your inventory
+
+    ansible all -i inventory  -u centos -k -m authorized_key -a "user=centos key=https://github.com/youraccount.keys"
+
+Note this makes use of your public key on github.  if you don't have a git hub account or a key pair on your git hub
+account, get one.
+
+the -k is needed cause I have to have it ask for a password at this point.
+
+now all commands can happen with out the password and -k option. Test with:
+
+    ansible all -i inventory -u centos -m ping
+
+You should get back a pong from each machine in your inventory.
+
+### Copy the /etc/host file over
+
+Add the nodes to /etc/hosts
+
+content is:
+
+        127.0.0.1 localhost localhost.localdomain localhost4 localhost4.localdomain4
+        ::1         localhost localhost.localdomain localhost6 localhost6.localdomain6
+        172.16.222.5 MyMac
+        172.16.222.6 control01
+        172.16.222.7 control02
+        172.16.222.8 control03
+        172.16.222.11 resource01
+        172.16.222.12 resource02
+        172.16.222.13 resource03
+        172.16.222.16 edge01
+        172.16.222.17 edge02
+
+Copy the /etc/host file over.
+
+    ansible all -i inventory -u centos --sudo --ask-sudo-pass -m copy -a "src=hosts dest=/etc/hosts"
+
+
+Might as well set the timezone just for grins.
+
+    ansible all -i inventory -u centos --sudo --ask-sudo-pass -m command -a "timedatectl set-timezone America/Los_Angeles"
+
+
+## Run It!
+
+You now are ready to run the playbook.  Change directory to the mantl root.  Your inventory should be there as well as
+the file `security_setup` (along with quite a few others).
+
+Run the security-setup script:
+
+    ./security-setup
+
+It asks for one admin password. At the end of that run there will be a `security.yml` file.  It will have the password
+you entered and a lot of keys and such that are used in the installs and in the installed software.
+
+The file you will be running is `terraform.sample.yml`.  Since you created your own inventory and didn't use terraform,
+there are a few environment variables you need to set for your run.
+
+    ansible-playbook -u centos -i inventory \
+            -e provider=bare-metal \
+            -e consul_dc=dc1 \
+            -e docker-lvm-backed=true \
+            -e docker_lvm_data_volume_size="80%FREE" \
+            -e @security.yml  terraform.sample.yml >& bare-metal.log
+
+
+In another window tail -f  that log file to follow whats going on.
+
+The meaning of the parts of this command are as follows:
+
+ * ansible-playbook -u centos -i inventory
+   * run the ansible play book as centos user against the inventory found in the ./inventory file.
+ * -e provider=bare-metal
+   * The "provider" is not terraform for a cloud, but rather bare-metal. If the inventory had been generated
```

Comment:
The provider is never terraform -- it is one of "gce", "aws", etc. Maybe say "The provider is not one of the cloud providers, but rather bare-metal."